containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -256,8 +256,8 @@ class UnorderedMap {
 
   using insert_result = UnorderedMapInsertResult;
 
-  using HostMirror =
-      UnorderedMap<Key, Value, host_mirror_space, Hasher, EqualTo>;
+  using host_mirror_type =
+      UnorderedMap<Key, Value, host_mirror_device, Hasher, EqualTo>;
```

Comment:
We may need to hide this change (i.e. changing `host_mirror_space` to `host_mirror_device`) behind a deprecation guard. We are changing types here, which means that in principle it's a breaking change.

Reply:
Fixed